React/game/src/app/screens/Login/components/Login/layout.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+import styles from './styles.scss';
+
+const validate = values => {
+  const errors = {};
+  if (!values.username) {
+    errors.username = 'Required';
+  } else if (values.username.length > 15) {
+    errors.username = 'Must be 15 characters or less';
+  }
+  if (!values.password) {
+    errors.password = 'Required';
+  } else if (values.password.length < 8) {
+    errors.password = 'invalid Password';
+  }
+  return errors;
+};
+
+const renderField = ({ input, label, type, meta: { touched, error, warning } }) => (
```

Comment:
Extract this component in another file. Remember, one component per file.

Reply:
👍